keps/sig-node/2902-cpumanager-distribute-cpus-policy-option/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: CPUManager Policy Option to Distribute CPUs Across NUMA Nodes Instead of Packing Them
+kep-number: 2902
+authors:
+  - "@klueska"
+owning-sig: sig-node
+participating-sigs: []
+status: implementable
+creation-date: "2021-08-26"
+reviewers:
+  - "@fromani"
+approvers:
+  - "@sig-node-leads"
+see-also:
+  - "keps/sig-node/2625-cpumanager-policies-thread-placement"
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.23"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.23"
+  beta: "v1.24"
+  stable: "v1.25"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: "CPUManagerPolicyExperimentalOptions"
```

Comment:
the option and kep makes sense, but what is less clear to me is ho this feature goes from alpha->beta->stable is all options are gated by `CPUManagerPolicyExperimentalOptions` gate.  the new options appear distinct from any feature gate and appear to have one of two states: present or hidden.

It seems feature gate `CPUManagerPolicyExperimentalOptions` is just in a permanent alpha state.

Reply:
For reference, we discussed this in more detail here, leading to the eventual approval without modification:
https://kubernetes.slack.com/archives/C0BP8PW9G/p1631123745114500

Reply:
Yes, it is different, as you said. Maybe other folks will object, but it seems like a painful state of affairs to feature gate. Probably should discuss in enhancements subproject.